Add follow button to the participatory spaces menu bar

entantoencuanto opened the pull request:
<!--
NOTE: We are in the middle of a big redesign of the frontend.
That could mean that your PR will not get through on the next few months.
Please see https://github.com/decidim/decidim/discussions/9512
-->

#### :tophat: What? Why?

This PR:
* Includes a follow button in the participatory spaces menu bar using a desktop and a mobile version.
* Adds some test

#### :pushpin: Related Issues
*Link your PR to an issue*
- Fixes #11204

#### Testing

Visit the landing page of the participatory space, see the menu bar, click on the button

### :camera: Screenshots

![Screenshot from 2023-09-25 21-26-37](https://github.com/decidim/decidim/assets/446459/9f90c9c6-d1ab-44c9-a6cd-8e9371611a72)

:hearts: Thank you!

alecslupu reviewed (changes requested):
We cannot add the follow button from the last activity block, since there is no such block in Voting, Conferences or Initiatives.

alecslupu commented:
@furilo , @ferblape do we have an update regarding this?

ferblape commented:
It's going to be reviewed this week by Edu

entantoencuanto commented:
I have moved the follow button to the menu bar actions space in assemblies, conferences, votings, initiatives and participatory processes and removed redundant buttons from activities content block for assemblies and participatory processes and other one remaining in initiatives

![Screenshot from 2023-09-25 21-26-37](https://github.com/decidim/decidim/assets/446459/9f90c9c6-d1ab-44c9-a6cd-8e9371611a72)

alecslupu commented:
> I have moved the follow button to the menu bar actions space in assemblies, conferences, votings, initiatives and participatory processes and removed redundant buttons from activities content block for assemblies and participatory processes and other one remaining in initiatives

This is Great! We are one step closer to merge this. One step to go, and we can merge it... There is no "follow button" visible on mobile ( phone ).

andreslucena commented:
@entantoencuanto I've changed the body and the title to reflect the latest changes

entantoencuanto commented:
Follow button added in mobile version
![Screenshot from 2023-09-26 12-54-32](https://github.com/decidim/decidim/assets/446459/b3622388-2970-401d-a93f-505032dd7e60)
